lib/routes/gmu/yjs.ts

```diff
@@ -0,0 +1,209 @@
+// | xzzx | xsxz | 下载中心学生下载 |
+| xzzx | xwxz | 下载中心学位下载 |`,
+    radar: [
+        {
+            source: ['yjs.gmu.cn/:type/:subtype.htm', 'yjs.gmu.cn/'],
+            target: '/yjs/:type/:subtype',
+        },
+    ],
+    handler,
+};
+
+export async function handler(ctx) {
+    const { type, subtype } = ctx.req.param();
+
+    if (!sections[type] || !sections[type][subtype]) {
+        throw new Error('Invalid type or subtype');
+    }
+
+    const { title, path } = sections[type][subtype];
+    const baseUrl = 'https://yjs.gmu.cn';
+    const link = baseUrl + path;
+
+    const response = await got(link, {
+        https: {
+            rejectUnauthorized: false,
+        },
+        timeout: 10000,
+        retry: 3,
+    });
+
+    const $ = load(response.data);
+
+    // 更新选择器以匹配研究生院网站的实际结构
+    const list = $('.n_listxx1 li');
+
+    if (list.length === 0) {
+        throw new Error('No content found. The page structure might have changed.');
+    }
+
+    // Extract item processing logic to a separate function to reduce complexity
+    async function processYjsItem(item, type, subtype, baseUrl, link) {
+        const element = $(item);
+
+        // 针对不同栏目适配不同结构
+        let a, dateText, itemTitle, description;
+        if (['bmgk', 'xzzx'].includes(type)) {
+            // 适配部门概况、下载中心等栏目
+            a = element.find('a').first();
+            dateText = element.find('span.time').text().trim() || '';
+            itemTitle = a.text().trim();
+            description = element.find('p').text().trim() || '';
+        } else if (type === 'xsgz' && subtype === 'xshd') {
+            // 适配学生活动特殊结构
+            a = element.find('a').first();
+            dateText = element.find('span.time').text().trim() || '';
+            itemTitle = a.text().trim();
+            description = element.find('p').text().trim() || '';
+        } else {
+            // 默认结构
+            a = element.find('h2 a').first();
+            dateText = element.find('h2 span.time').text().trim();
+            itemTitle = a.text().trim();
+            description = element.find('p').text().trim();
+        }
+
+        const href = a.attr('href');
+
+        if (!href || !itemTitle) {
+            return null;
+        }
+
+        const pubDate = parseDate(dateText);
+        if (!pubDate) {
+            return null;
+        }
+
+        let fullLink = href;
+        if (href.startsWith('/')) {
+            fullLink = new URL(href, baseUrl).href;
+        } else if (!href.startsWith('http')) {
+            const currentPath = new URL(link).pathname;
+            const basePath = currentPath.slice(0, Math.max(0, currentPath.lastIndexOf('/') + 1));
+            fullLink = new URL(basePath + href, baseUrl).href;
+        }
+
+        try {
+            const contentResponse = await got(fullLink, {
+                https: {
+                    rejectUnauthorized: false,
+                },
+                timeout: 10000,
+                retry: 3,
```

Comment:
Do not override RSSHub's config https://github.com/DIYgod/RSSHub/blob/59e7ea876df12bbaf8a4f72d5de12f99f8f9cff3/lib/config.ts#L459-L460